app/javascript/includes/navbar.js

```diff
@@ -1,4 +1,5 @@
-$(document).ready(function(){
+$( document ).on('turbolinks:load', function() {
+ $(document).ready(function(){
```

Comment:
.ready is inside .on? that seems to be odd. what if the user reloads the page will this still work?

Reply:
It is resolved in [bug/discussion-close](https://github.com/MilesUpHQ/reddit-clone-rails/tree/bug/discussion-close)